android/build.gradle

```diff
@@ -288,3 +288,27 @@ tasks.register("copyCodegenArtifacts") {
 if (isGHExampleApp() && isNewArchitectureEnabled()) {
     tasks.generateCodegenArtifactsFromSchema.finalizedBy('copyCodegenArtifacts')
 }
+
+tasks.register("checkIntegrityBetweenArchitectures") {
+    group 'Verification tasks'
+    description 'Task to check integrity between fabric and paper architecture in terms of codegen generated interfaces/delegates'
+
+    if (isGHExampleApp()) {
+        return
+    }
+
+    def absoluteCodegenArtifactsPaperDestination = "../${project.property('codegenArtifactsPaperDestination')}"
+    def absoluteCodegenArtifactsSource = "../${project.property('codegenArtifactsSource')}"
+
+    def existingFiles = fileTree(absoluteCodegenArtifactsPaperDestination).matching {
+        include '**/*.java'
+    }
+
+    existingFiles.forEach { existingFile -> 
+        def generatedFile = new File("${absoluteCodegenArtifactsSource}/${existingFile.name}")
+
+        if (existingFile.text != generatedFile.text) {
+            throw new RuntimeException("[react-native-gesture-handler] The source of ${existingFile.name} does not match with the one generated by codegen. Please check if you commited changes produced by copyCodegenArtifacts task.")
+        }
+    }
+}
```

Comment:
New line

Reply:
Fixed in https://github.com/software-mansion/react-native-gesture-handler/pull/2933/commits/b91141aaec7ef089ad793e969e3643a98721a98a